dmutils/content_loader.py

```diff
@@ -9,12 +9,16 @@
 from .config import convert_to_boolean, convert_to_number
 
 
-class ContentBuilder(object):
+class ContentNotFoundError(Exception):
+    pass
+
+
+class ContentManifest(object):
     """An ordered set of sections each made up of one or more questions.
 
     Example::
         # Get hold of a section
-        content = ContentBuilder(sections)
+        content = ContentManifest(sections)
         section = content.get_section(section_id)
 
         # Extract data from form data
```

Comment:
Not really to do with this pull request but this docstring style doesn't match the style used on `ContentLoader`